libvmi/driver/kvm/include/asm/kvmi.h

```diff
@@ -0,0 +1,124 @@
+/* SPDX-License-Identifier: GPL-2.0 WITH Linux-syscall-note */
```

Comment:
Same here. Usually ABI headers are exempt from license restrictions to avoid issues (at least thats the case with Xen). Isn't that the case here too?

Reply:
my earlier comment applies here also.

Reply:
@Wenzel Thank you for addressing